src/main/scala/zio/ftp/settings.scala

```diff
@@ -0,0 +1,107 @@
+package zio.ftp
+
+import java.net.Proxy
+
+import net.schmizz.sshj.{ Config => SshConfig, DefaultConfig => DefaultSshConfig }
+import org.apache.commons.net.ftp.FTPClient
+
+sealed trait credentials {
+
+  sealed trait FtpCredentials {
```

Comment:
`case class` instead of trait?

Reply:
i replace the whole stuff with simpler approach
```scala
object settings {
   type FtpCredentials = (String, String)
   def credentials(username: String , password: String) = username -> password

   final case class SFtpSettings(
    host: String,
    port: Int,
    credentials: FtpCredentials,
    strictHostKeyChecking: Boolean,
    knownHosts: Option[String],
    sftpIdentity: Option[SftpIdentity],
    sshConfig: SshConfig){
    val username = credentials._1
    val password = credentials._2
  }
}
```